src/components/common/menu/Header.tsx

```diff
@@ -0,0 +1,69 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { useEffect,useState } from 'react';
+
+type UserType = {
+  id: number;
+};
+
+type HeaderProps = {
+  user?: UserType;
+}
+
+export default function Header({ user, ...props }: HeaderProps) {
+  const [isSearchVisible, setSearchVisible] = useState(false);
+  const [windowWidth, setWindowWidth] = useState(0);
+  const hamburger = '/icons/hamburger.svg';
+  const logo = '/icons/logo.svg';
+  const search = '/icons/search.svg';
+
+  const toggleSearch = () => {
+    setSearchVisible(!isSearchVisible);
+  };
+
+  useEffect(() => {
+    const updateWindowWidth = () => {
+      setWindowWidth(window.innerWidth);
+    };
+    updateWindowWidth();
+    window.addEventListener('resize', updateWindowWidth);
+    return () => {
+      window.removeEventListener('resize', updateWindowWidth);
+    };
+  }, []);
+
+  return (
+    <div className='flex h-[7rem] w-[100%] min-w-[37.5rem] flex-row justify-between bg-[#1c1c22] align-middle md:h-[8rem] lg:h-[10rem]'>
+      <Image src={hamburger} alt='side menu' width={24} height={24} className='ml-[2rem] block md:hidden lg:hidden' />
```

Comment:
```suggestion
      <Image src={hamburger} alt='side menu' width={24} height={24} className='ml-[2rem] block md:hidden' />
```
min size로 적용되서 md:hidden까지만 작성해도 괜찮은거같아요!

Reply:
요건 완전히 간과하고 있던 포인트네요 감사해요 !!